deps/blas.mk

```diff
@@ -46,7 +46,7 @@ $(BUILDDIR)/$(OPENBLAS_SRC_DIR)/build-compiled: | $(BUILDDIR)/objconv/build-comp
 endif
 endif
 
-OPENBLAS_FFLAGS :=
+OPENBLAS_FFLAGS := $(FFLAGS) $(JFFLAGS)
 OPENBLAS_CFLAGS :=
```

Comment:
maybe save the VERBOSE log of a build with vs without this change to separate files and diff them? there might be an upstream bug here with flags not getting sent everywhere they should